tests/WebAppCallsWebApiCallsGraph/Client/Controllers/TodoListController.cs

```diff
@@ -6,21 +6,21 @@
 using Microsoft.Identity.Web;
 using System.Threading.Tasks;
 using TodoListClient.Services;
-using TodoListService.Models;
+using TodoList.Models;
 
 namespace TodoListClient.Controllers
 {
     public class TodoListController : Controller
     {
         private ITodoListService _todoListService;
 
-        public TodoListController(ITodoListService todoListService)
+        public TodoListController(IDownstreamWebApi downstreamWebApi)
         {
-            _todoListService = todoListService;
+            _todoListService = new TodoListService(downstreamWebApi);
```

Comment:
I'm not too satisfied it, but I did not yet manage to inject the DownstreamApiService.
If we decided to go with this design, we'll have to fix that.